src/peergos/server/tests/simulation/Simulator.java

```diff
@@ -263,13 +270,18 @@ public Simulator(int opCount, Random random, long meanFileLength,
         this.index = new FileSystemIndex(random);
     }
 
-    private boolean read(String user, Path path) {
+    private void readFile(String user, Path path) {
         FileSystem testFileSystem = fileSystems.getTestFileSystem(user);
         FileSystem referenceFileSystem = fileSystems.getReferenceFileSystem(user);
+        referenceFileSystem.read(path);
+        testFileSystem.read(path);
+    }
 
-        byte[] refBytes = referenceFileSystem.read(path);
-        byte[] testBytes = testFileSystem.read(path);
-        return Arrays.equals(refBytes, testBytes);
```

Comment:
Why is  this not comparing the bytes anymore?

Reply:
The return value was never checked previously anyway. The contents is now checked as part of verify()